code/modules/mob/living/simple_animal/friendly/mouse.dm

```diff
@@ -49,38 +49,43 @@
 	can_pull_size = 1
 	can_pull_mobs = MOB_PULL_NONE
 
-/mob/living/simple_animal/mouse/Life()
-	..()
+	var/decompose_time = 18000
 
-	if(client)
-		//Player-animals don't do random speech normally, so this is here
-		//Player-controlled mice will still squeak, but less often than NPC mice
-		if (stat == CONSCIOUS && prob(speak_chance*0.1))
-			squeak_soft(0)
-
-		if(is_ventcrawling == 0)
-			sight = SEE_SELF // Returns mouse sight to normal when they leave a vent
-
-		if (squeals < maxSqueals)
-			var/diff = world.time - last_squealgain
-			if (diff > 600)
-				squeals++
-				last_squealgain = world.time
-
-	if(!ckey && stat == CONSCIOUS && prob(0.5))
-		stat = UNCONSCIOUS
-		icon_state = "mouse_[body_color]_sleep"
-		wander = 0
-		speak_chance = 0
-		//snuffles
-	else if(stat == UNCONSCIOUS)
-		if(ckey || prob(1))
-			stat = CONSCIOUS
-			icon_state = "mouse_[body_color]"
-			wander = 1
-			speak_chance = initial(speak_chance)
-		else if(prob(5))
-			audible_emote("snuffles.")
+/mob/living/simple_animal/mouse/Life()
+	if(..())

```

Comment:
this check and indent saves a little cpu time on dead mice.

When writing this originally, i forgot that life returns 0 if the animal is dead.